### PR TITLE
About Screen

### LINKED ISSUE #10: About Screen
Shows Pumpkin Reader & Pumpkinz informations.

### PR DESCRIPTION
**Pull request by wiseodd**
Closes #10

### CODE REVIEW

File: app/src/main/java/io/pumpkinz/pumpkinreader/MainActivity.java [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.Toolbar;
+import android.util.Log;
 import android.view.MenuItem;
 import android.view.WindowManager;
 
 
 public class MainActivity extends AppCompatActivity {
 
+
```
Comment (timotiusnc) [inline app/src/main/java/io/pumpkinz/pumpkinreader/MainActivity.java]: one line too many?

File: app/src/main/java/io/pumpkinz/pumpkinreader/AboutActivityFragment.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+public class AboutActivityFragment extends Fragment {
+
+    public AboutActivityFragment() {
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.fragment_about, container, false);
+    }
```
Comment (timotiusnc) [inline app/src/main/java/io/pumpkinz/pumpkinreader/AboutActivityFragment.java:19]: one line short?

**Comment by timotiusnc** [comment]
Should we add [RxAndroid](https://github.com/ReactiveX/RxAndroid)?

Unable to merge, btw.

**Comment by wiseodd** [comment]
@timotiusnc updated

**Pull request merged**